jsonnet/utils.libsonnet

```
@@ -196,6 +196,9 @@
     packageNameTemplate: '@trunkio/launcher',
     depNameTemplate: 'trunk-io/launcher',
   },
+  bitwardenClients: $.prefixRegexManager('bitwarden/clients', 'cli-') + {
+    packageNameTemplate: 'bitwarden/clients',
+  },
```

```suggestion
  bitwardenClients: $.prefixRegexManager('bitwarden/clients', 'cli-'),
```

I guess `packageNameTemplate` isn't required as packageNameTemplate is equivalent to depNameTemplate.

---

- https://github.com/suzuki-shunsuke/test-renovate/blob/1c21aa9655fcafe2ef10bc380ec0b699897ed429/renovate.json5#L8-L26
- https://github.com/suzuki-shunsuke/test-renovate/pull/301